docs/api/api.md

```diff
@@ -180,6 +180,37 @@ You do not need to care about the intrinsics, since the returned `id` and `proof
 
 A nice example of using this endpoint via our SDK can be found [here](../dev/developer-guides/bridging/l2-l1.md).
 
+::: warning
+
+`zks_getL2ToL1MsgProof` endpoint will be deprecated because proofs for L2 to L1 messages can also be fetched from `zks_getL2ToL1LogProof`.
+
+:::
+
+### `zks_getL2ToL1LogProof`
+
+Given a transaction hash, and an index of the L2 to L1 log produced within the transaction, it returns the proof for the corresponding L2 to L1 log.
+
+The index of the log that can be obtained from the transaction receipt (it includes a list of every log produced by the transaction).
+
+### Input parameters
+
+| Parameter | Type      | Description                                                                               |
+| --------- | --------- | ----------------------------------------------------------------------------------------- |
+| tx_hash   | `bytes32` | Hash of the L2 transaction the L2 to L1 log was produced within.                                   |
+| l2_to_l1_log_index| `undefined | number` | The Index of the L2 to L1 log in the transaction. |
```

Comment:
nit: for me it shows `undefined | number` in 2 lines, which looks bad, if you know how to fix it, please do, if not then let's leave

Reply:
good point... most likely there is a way to escape the character inside the table - https://stackoverflow.com/questions/49809122/vertical-bar-symbol-within-a-markdown-table

Reply:
Unfortunately, it doesn't seem to work.